sys/mips/malta/malta_machdep.c

```diff
@@ -277,6 +277,44 @@ malta_cpu_freq(void)
 	return (platform_counter_freq);
 }
 
+
+/* Parse cmd line args as env - copied from beri_machdep. */
+static void
+_parse_bootargs(char *cmdline)
+{
+	char *n, *v;
+
+	while ((v = strsep(&cmdline, " \n")) != NULL) {
```

Comment:
Is the intent here to parse based on all whitespace -- if so, should '\t' also be in the strsep() separator argument?